### PR TITLE
l10n support collection static files

### PR DESCRIPTION
**Pull request by tigershi**
add the support MD file feature

### CODE REVIEW

File: g11n-ws/vip-manager-l10n/src/main/java/com/vmware/l10n/source/service/impl/SourceServiceImpl.java [hunk 1]
```diff
@@ -84,40 +85,52 @@ public void writeSourceToCachedFile() {
...
+			addNewStringSource(strDTO, catcheKey, key, source, comment, sourceFormat);
 		} else {
-			updateStringSource(comp, key, source, comment);
+			updateStringSource(comp, key, source, comment, sourceFormat);
 		}
+        if(!StringUtils.isEmpty(sourceFormat) && (MD.equals(sourceFormat) || HTML.equals(sourceFormat))) {
+        	return true;
+        }else {
+        	return false;
+        }
```
Comment (linr211) [inline g11n-ws/vip-manager-l10n/src/main/java/com/vmware/l10n/source/service/impl/SourceServiceImpl.java:108]: move these verification to a common method

File: g11n-ws/vip-manager-l10n/src/main/java/com/vmware/l10n/source/service/impl/SourceServiceImpl.java [hunk 2]
```diff
@@ -34,6 +33,8 @@
 public class SourceServiceImpl implements SourceService {
 	
 	private final static Logger LOGGER = LoggerFactory.getLogger(SourceService.class);
+	private final static String MD = "MD";
+	private final static String HTML = "HTML";
```
Comment (linr211) [inline g11n-ws/vip-manager-l10n/src/main/java/com/vmware/l10n/source/service/impl/SourceServiceImpl.java:37]: define a string array for the content types.

Comment (tigershi) [reply g11n-ws/vip-manager-l10n/src/main/java/com/vmware/l10n/source/service/impl/SourceServiceImpl.java]: done

File: g11n-ws/vip-manager-l10n/src/main/java/com/vmware/l10n/source/service/impl/SourceServiceImpl.java (hunk 1 above)
Comment (tigershi) [reply g11n-ws/vip-manager-l10n/src/main/java/com/vmware/l10n/source/service/impl/SourceServiceImpl.java]: done

File: g11n-ws/modules/md-restful-api/src/main/java/com/vmware/vip/api/rest/APIV2.java [hunk 3]
```diff
@@ -23,6 +23,7 @@
 	public static final String KEY_TRANSLATION_GET        = API_TRANSLATOIN + "/products/{"+APIParamName.PRODUCT_NAME+"}/versions/{"+APIParamName.VERSION2+"}/locales/{"+APIParamName.LOCALE+"}/components/{"+APIParamName.COMPONENT+"}/keys/{"+APIParamName.KEY2+"}";
 	public static final String KEY_TRANSLATION_POST       = API_TRANSLATOIN + "/products/{"+APIParamName.PRODUCT_NAME+"}/versions/{"+APIParamName.VERSION2+"}/locales/{"+APIParamName.LOCALE+"}/components/{"+APIParamName.COMPONENT+"}/keys/{"+APIParamName.KEY2+"}";
 	public static final String KEY_SET_POST               = API_TRANSLATOIN + "/products/{"+APIParamName.PRODUCT_NAME+"}/versions/{"+APIParamName.VERSION2+"}/locales/{"+APIParamName.LOCALE+"}/components/{"+APIParamName.COMPONENT+"}/keys";
+	public static final String KEY_SET_GET               = API_TRANSLATOIN + "/products/{"+APIParamName.PRODUCT_NAME+"}/versions/{"+APIParamName.VERSION2+"}/locales/{"+APIParamName.LOCALE+"}/components/{"+APIParamName.COMPONENT+"}/keys";
```
Comment (linr211) [inline g11n-ws/modules/md-restful-api/src/main/java/com/vmware/vip/api/rest/APIV2.java:26]: please separate this new API to another PR, we need to further discuss it.

Comment (tigershi) [reply g11n-ws/modules/md-restful-api/src/main/java/com/vmware/vip/api/rest/APIV2.java]: done

**Review by jin-jasper — APPROVED** [review]
